### DISCUSSION TITLE
「PWN」【香山杯2023】Writeup WP 复现 | ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#5] [Announcements]
# 「PWN」【香山杯2023】Writeup WP 复现 | ネコのメモ帳

两题简单 PWN，单纯更新一下，记录一下 Python 调试

https://n.ova.moe/reproducing/2023/10/17/%E9%A6%99%E5%B1%B1%E6%9D%AF2023

**Comment by MuelNova** [comment]
其实似乎可以使用 gdb 指令
```sh
set breakpoint pending on
```
之后直接打断点
